src/global.css

```diff
@@ -19,6 +19,17 @@ body {
   height: var(--navbar-height);
 }
 
+nav {
+  position: fixed;
+  width: 100vw;
+  top: 0px;
+  z-index: 100;
+}
+
+.m-t-navBarHeight {
```

Comment:
I think this might be a good chance to show how to export react components directly https://github.com/react-microfrontends/styleguide/blob/b22fe17015e8aee01afd48f674734f6fe21ae779/src/react-mf-styleguide.js#L4

It's something I think would be good to show in this demo repo. Perhaps we have a `<MainContent>` component that simply applies the margin top?

Reply:
I was going to add that with a button component in the future. For layout I'd like to encourage a pure CSS approach as much as possible

Reply:
fair enough